introduction.md

```diff
@@ -1,5 +1,12 @@
 Favorite path: iOS
 
+##Skills List
+*Git
+*iOS
+*Ruby
+*Javascript
+
+
 Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque varius, nulla commodo consequat lobortis, nulla tortor molestie erat, id pretium nulla est sit amet lectus. Vivamus dapibus arcu ante, sit amet eleifend ex gravida in. Pellentesque quis dui sed erat rutrum efficitur sed malesuada enim. Sed at velit nisl. Sed vulputate arcu leo, et placerat mauris viverra ut. Morbi leo enim, tincidunt ac commodo eu, dignissim eget nunc. Nulla varius massa sit amet porta ultricies. Integer nec finibus massa, egestas elementum leo. Phasellus varius ante metus, non malesuada sapien hendrerit at. In imperdiet neque purus, et porttitor libero pharetra vitae.
 
 Aenean et gravida lacus, ut malesuada ipsum. Suspendisse condimentum ligula vel metus eleifend, vitae pretium massa interdum. Fusce luctus tincidunt molestie. Nunc justo nisi, lobortis eu venenatis at, sagittis ac libero. Aenean felis elit, feugiat at blandit at, euismod a nulla. Aenean diam nunc, posuere id sagittis eu, malesuada ut felis. Suspendisse interdum ultrices eleifend. Pellentesque bibendum suscipit quam, in accumsan nunc euismod eget. Nam dapibus augue id venenatis consectetur. Aenean cursus quam ac nisi volutpat, ut elementum libero venenatis. Morbi ac sapien ornare, dictum mi ut, aliquet arcu. Integer bibendum urna elit, quis tristique leo consequat at.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?